test/integration/tests/staking.ts

```diff
@@ -0,0 +1,559 @@
+import { AutoRewardsDistributor, TribalChief } from '@custom-types/contracts';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { BigNumber, Contract } from 'ethers';
+import hre, { ethers } from 'hardhat';
+import { NamedAddresses, NamedContracts } from '@custom-types/types';
+import { expectApprox, getImpersonatedSigner, resetFork, time } from '@test/helpers';
+import proposals from '@test/integration/proposals_config.json';
+import { TestEndtoEndCoordinator } from '../setup';
+import { forceEth } from '@test/integration/setup/utils';
+
+const toBN = ethers.BigNumber.from;
+
+before(async () => {
+  chai.use(CBN(ethers.BigNumber));
+  chai.use(solidity);
+  await resetFork();
```

Comment:
Does this before block execute before the top level describe block starts?

Reply:
Yes